tests/introspection/ApiTypoTest.cs

```diff
@@ -737,15 +737,44 @@ bool IsObsolete (MemberInfo mi)
 			return IsObsolete (mi.DeclaringType);
 		}
 
+		[Test]
+		public virtual void AttributeTypoTest ()
+		{
+			var types = Assembly.GetTypes ();
+			int totalErrors = 0;
+			foreach (Type t in types)
+				AttributeTypo (t, ref totalErrors);
+
+			Assert.IsTrue ((totalErrors == 0), "Attributes have {0} typos!", totalErrors);
```

Comment:
This seems a bit simpler, and provides the same information:

```suggestion
			Assert.AreEqual (0, totalErrors, "Attributes have typos!");
```

Reply:
agreed - that's a copy/paste of the original test